.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+name: Build
+on: 
+  pull_request:
```

Comment:
`push` and `tag`? See https://github.com/rancher/csp-adapter/blob/release/v2.9/.drone.yml#L19-L21

Reply:
GHA events triggers workflows, not single steps as drone does. That's why we have build and release yamls to separate what should run on pushes and what should run on tag events.

Reply:
I see. In that case, I guess we need to add a separate workflow for `push`?

Reply:
That was the approach I was using at first. But if we use another workflow for push we will need release artifacts to share data between workflows, which is discouraged.